cabal.project

```diff
@@ -2,13 +2,8 @@ packages: . ormolu-live
 
 constraints: ormolu +dev
 
--- (temporarily) necessary for ormolu-live
--- (will be fixed by next ghc-lib-parser release)
--- This removes a redundant dependency on hpc, which
--- is incompatible with haskell.nix+GHCJS.
 source-repository-package
   type: git
-  location: https://github.com/amesgen/stuff
-  tag: 5d173a747118880ded6568001c8d0d56b57ac69f
-  subdir: ghc-lib-parser-9.0.1.20210324
-  --sha256: 0qh07100qvxf0bh2rz64n21cz92wc1pj4niy3xs027l8xv52syz2
+  location: https://github.com/amesgen/ghc-syntax-highlighter
```

Comment:
I released a new version (0.0.8.0) of `ghc-syntax-highlighter` yesterday, do you know how long it takes for `haskell.nix` to pick that up?

Reply:
I was just about to push a commit to remove this; haskell.nix updates its Hackage snapshot daily.